tests/queries/0_stateless/01710_projection_fetch.sql

```diff
@@ -32,9 +32,9 @@ select count() from system.projection_parts where table = 'tp_2' and name = 'pp'
 show create table tp_2;
 
 alter table tp_1 drop projection pp;
-select count() from system.projection_parts where table = 'tp_2' and name = 'pp' and active;
+select count() from system.projection_parts where table = 'tp_2' and name = 'pp' and active;                          
+system sync replica tp_2;
```

Comment:
I thought we don't need this because `drop projection` is a mutation and we already set `mutations_sync = 2`

Reply:
The test is still flaky.
Probably, `mutations_sync = 2` is broken for now ...

Reply:
Still flacky :(
I suspect there is a natural race condition for `system.projection_parts` ...